### ISSUE TITLE
Has these models already implemented support for group normalization?

### ISSUE THREAD

**Issue by lartpang** [pytorch/vision#1072]
https://github.com/pytorch/vision/blob/8350645b680b5dc0ef347de82deea5ae3f8ca3dc/torchvision/models/resnet.py#L85

When I see these code, I thought the model had implemented the support about 'Group Normalization`...

https://github.com/pytorch/vision/blob/8350645b680b5dc0ef347de82deea5ae3f8ca3dc/torchvision/models/resnet.py#L156-L158

**Comment by fmassa** [comment]
Yes, the model support Group Norm, but there are no pre-trained models for those.

Group Norm generally doesn't improve classification accuracy, but is better when training on small-batch regime like for object detection

**Comment by lartpang** [comment]
OK, thanks for your reply. Group Norm is suitable for me, :smiley: .

**Comment by fmassa** [comment]
We currently have no plans to add pre-trained models with group norm for classification, at least for the next release of torchvision.

**Comment by netw0rkf10w** [comment]
@fmassa Could you tell me how to instantiate a model with GroupNorm? The following doesn't work:

```python
torchvision.models.resnet50(norm_layer=torch.nn.GroupNorm)
```

Thanks.

**Comment by netw0rkf10w** [comment]
Solved:
```python
class GroupNorm32(torch.nn.GroupNorm):
    def __init__(self, num_channels, num_groups=32, **kargs):
        super().__init__(num_groups, num_channels, **kargs)
```

**Comment by lartpang** [comment]
I have found the repo which has some models with groupnorm:

https://github.com/ppwwyyxx/GroupNorm-reproduce
